### PR TITLE
fix: iframe input hook (#990)

### PR DESCRIPTION
**Pull request by luocongqiu**

### CODE REVIEW

File: packages/rrweb/src/record/observer.ts [hunk 1]
```diff
@@ -428,18 +428,25 @@ function initInputObserver({
   const handlers: Array<
     listenerHandler | hookResetter
   > = events.map((eventName) => on(eventName, eventHandler, doc));
-  const propertyDescriptor = Object.getOwnPropertyDescriptor(
-    HTMLInputElement.prototype,
+
+  const currentWindow = doc.defaultView;
```
Comment (Mark-Fenng) [inline packages/rrweb/src/record/observer.ts:432]: The code all looks good to me. Only a code format (using prettier) is required.

**Review by Mark-Fenng — CHANGES_REQUESTED** [review]
@luocongqiu Thanks for your contribution both on the issue and pull request. Your pull request looks great.

**Review by Yuyz0112 — APPROVED** [review]
good catch!

**Review by Mark-Fenng — APPROVED** [review]

File: packages/rrweb/src/record/observer.ts [hunk 2]
```diff
@@ -449,7 +455,7 @@ function initInputObserver({
             // mock to a normal event
             eventHandler({ target: this as EventTarget } as Event);
           },
-        }),
+        }, false, currentWindow),
```
Comment (Mark-Fenng) [inline packages/rrweb/src/record/observer.ts:458]: @luocongqiu Seems like there's also a format problem.
Please execute `yarn prettier --write '**/*.{ts,md}'` to fix them all.

Comment (luocongqiu) [reply packages/rrweb/src/record/observer.ts]: @Mark-Fenng thanks，I've formatted the code

**Pull request merged**